app/controllers/spaces_controller.rb

```diff
@@ -5,11 +5,25 @@ class SpacesController < ApplicationController
 
   #GET /spaces
   #GET /spaces.json
+  #TODO(matthew): params['strings'] or params[:symbols]?
   def index
     if params['user_id']
```

Comment:
This should at least be split up into two other methods, maybe ``user_index`` and ``organization_index`` or ``user_spaces``.